packages/ui/vite.config.ts

```diff
@@ -0,0 +1,42 @@
+import { defineConfig } from 'vite'
+import Windi from 'vite-plugin-windicss'
+import Vue from '@vitejs/plugin-vue'
+import Icons from 'unplugin-icons/vite'
+import Svg from 'vite-svg-loader'
+import path from 'path'
+
+function resolve(...args: string[]): string {
+  return path.resolve(__dirname, ...args)
+}
+
+export default defineConfig({
+  resolve: {
+    alias: {
+      '@': resolve('src'),
+    },
+  },
+  plugins: [
+    Windi({
+      // explicit path in case when cwd is not the `__dirname`
+      config: resolve('windi.config.ts'),
+    }),
+    Vue(),
+    Icons(),
+    Svg(),
+  ],
+  build: {
+    cssCodeSplit: false,
+    minify: false,
+    chunkSizeWarningLimit: 2_000,
```

Comment:
Sorry is everything ok with `chunkSizeWarningLimit` value?

Reply:
Yes. It produces warnings about huge size of generated CSS bundle. It is about 500kb+ because of included minified `json-editor` CSS.